cmd/helm/template.go

```diff
@@ -154,3 +198,58 @@ func newTemplateCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 
 	return cmd
 }
+
+// write the <data> to <output-dir>/<name>. <append> controls if the file is created or content will be appended
+func writeToFile(outputDir string, name string, data string, append bool) error {
+	outfileName := strings.Join([]string{outputDir, name}, string(filepath.Separator))
+
+	err := ensureDirectoryForFile(outfileName)
+	if err != nil {
+		return err
+	}
+
+	f, err := createOrOpenFile(outfileName, append)
+	if err != nil {
+		return err
+	}
+
+	defer f.Close()
+
+	_, err = f.WriteString(fmt.Sprintf("---\n# Source: %s\n%s\n", name, data))
+
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("wrote %s\n", outfileName)
```

Comment:
Is this left-over debugging output, or is this supposed to be here? If it is intended to be here, it should use `fmt.Fprintf()` and send the results to whatever the command declares as `out`. This allows other things to override STDOUT with another ouput.

Reply:
Thanks! This line is used to debug.I will polish it with `fmt.Fprintf()` and forgive me for  my bad english.😄

Reply:
Thanks! This line is used to debug.Is it better to use [`debug()`](https://github.com/helm/helm/blob/7f3339cb4eec9800cb09a46512050fb509480f7e/cmd/helm/helm.go#L55) instead of `fmt.FprintF()`? Forgive me for  my bad english.😄

Reply:
This line is aimed to print some help information  
```bash
➜  myapp helm template  --output-dir tes . --set ingress.enabled=true --no-hooks
wrote tes/myapp/templates/serviceaccount.yaml
wrote tes/myapp/templates/service.yaml
wrote tes/myapp/templates/deployment.yaml
wrote tes/myapp/templates/ingress.yaml
```